Bottom: f243dd0dad2d7e5ba06b4d51e131232367a53c06
Top:    8b3146ce840e65e14a9e5b9636a956be56835448
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-21 23:05:40 +0100

Refresh of make-rtree_t-type-available-to

---

diff --git a/src/global.h b/src/global.h
index a38fe57..4cf489a 100644
--- a/src/global.h
+++ b/src/global.h
@@ -53,12 +53,10 @@
 typedef struct BoxType BoxType, *BoxTypePtr;
 typedef struct polygon_st PolygonType, *PolygonTypePtr;
 typedef struct drc_violation_st DrcViolationType, *DrcViolationTypePtr;
-
-#include "hid.h"
-
 typedef struct rtree rtree_t;
 
-#include "polyarea.h" /* Needs rtree_t defined */
+#include "hid.h"
+#include "polyarea.h"
 
 #define _(S) (S)
 
diff --git a/src/polygon1.c b/src/polygon1.c
index 0bbfbd2..8170197 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -47,7 +47,6 @@
 #include	<string.h>
 
 #include "global.h"
-#include "polyarea.h"
 #include "rtree.h"
 #include "heap.h"
